proposals/solid-oidc-primer/index.bs

```diff
@@ -159,62 +159,78 @@ describes everything the client will need to know to authorize with Alice's spec
 
 Response Body:
 ```json
-{  
-   "issuer":"https://secureauth.example",
-   "authorization_endpoint":"https://secureauth.example/authorize",
-   "token_endpoint":"https://secureauth.example/token",
-   "userinfo_endpoint":"https://secureauth.example/userinfo",
-   "jwks_uri":"https://secureauth.example/jwks",
-   "registration_endpoint":"https://secureauth.example/register",
-   "response_types_supported":[  
-      "code",
-      "code token",
-      "code id_token",
-      "id_token",
-      "id_token token",
-      "code id_token token",
-      "none"
-   ],
-   "response_modes_supported":[  
-      "query",
-      "fragment"
-   ],
-   "grant_types_supported":[  
-      "authorization_code",
-      "implicit",
-      "refresh_token",
-      "client_credentials"
-   ],
-   "subject_types_supported":[  
-      "public"
-   ],
-   "id_token_signing_alg_values_supported":[  
-      "RS256",
-      "RS384",
-      "RS512",
-      "none"
-   ],
-   "token_endpoint_auth_methods_supported":[  
-      "client_secret_basic"
-   ],
-   "token_endpoint_auth_signing_alg_values_supported":[  
-      "RS256"
-   ],
-   "display_values_supported":[  
-
-   ],
-   "claim_types_supported":[  
-      "normal"
-   ],
-   "claims_supported":[  
-
-   ],
-   "claims_parameter_supported":false,
-   "request_parameter_supported":true,
-   "request_uri_parameter_supported":false,
-   "require_request_uri_registration":false,
-   "check_session_iframe":"https://secureauth.example/session",
-   "end_session_endpoint":"https://secureauth.example/logout"
+{
+    "issuer": "https://secureauth.example",
+    "authorization_endpoint": "https://secureauth.example/authorize",
+    "token_endpoint": "https://secureauth.example/token",
+    "userinfo_endpoint": "https://secureauth.example/userinfo",
+    "jwks_uri": "https://secureauth.example/jwks",
+    "registration_endpoint": "https://secureauth.example/register",
+    "response_types_supported": [
+        "code",
+        "code token",
+        "code id_token",
+        "id_token",
+        "id_token token",
+        "code id_token token",
+        "none"
+    ],
+    "response_modes_supported": [
+        "query",
+        "fragment"
+    ],
+    "grant_types_supported": [
+        "authorization_code",
+        "implicit",
+        "refresh_token",
+        "client_credentials"
+    ],
+    "subject_types_supported": [
+        "public"
+    ],
+    "id_token_signing_alg_values_supported": [
+        "RS256",
+        "RS384",
+        "RS512",
+        "none"
+    ],
+    "token_endpoint_auth_methods_supported": [
+        "client_secret_basic"
+    ],
+    "token_endpoint_auth_signing_alg_values_supported": [
+        "RS256"
+    ],
+    "claim_types_supported": [
+        "normal"
+    ],
+    "claims_supported": [
+        "sub",
+        "name",
+        "preferred_username",
```

Comment:
I don't think these claims need to be mentioned. Nothing wrong with them, but including them in the primer might indicate that they are somehow required/expected